src/Sarif.Driver/Sdk/PluginDriverCommand.cs

```diff
@@ -234,32 +238,38 @@ protected virtual void PostLogFile(IAnalysisContext globalContext)
 
             try
             {
-                using (var httpClient = new HttpClient())
+                var postUri = new Uri(globalContext.PostUri);
+                string outputFilePath = globalContext.OutputFilePath;
+                IFileSystem fileSystem = globalContext.FileSystem;
+
+                (bool, string) result = SarifLog.Post(postUri, outputFilePath, fileSystem, httpClient)
+                    .GetAwaiter()
+                    .GetResult();
+
+                // This reporting isn't sent through the 'globalContext.Loggers' property
+                // because the post operation occurs after the backing SARIF log has 
+                // already been finalized.
+                if (!globalContext.Quiet || result.Item1 == false)
                 {
-                    PostLogFile(globalContext.PostUri, globalContext.OutputFilePath, globalContext.FileSystem, httpClient)
-                        .GetAwaiter()
-                        .GetResult();
+                    Console.WriteLine(result.Item2);
+                }
+
+                if (result.Item1 == false && !result.Item2.Contains("skipped"))
+                {
+                    globalContext.RuntimeErrors |= RuntimeConditions.ExceptionPostingLogFile;
                 }
```

Comment:
After I added `RuntimeExceptions` below when there is unhandled Exception, I realized a question, 
here when there is no unhandled Exception but the server is not "working", should we set `RuntimeErrors` without setting `RuntimeExceptions` ? I use the last commit for this change, if not we can just revert the commit 7.